lib/ruby_llm/models.json

```diff
@@ -1746,6 +1746,26 @@
       ]
     }
   },
+  {
+    "id": "o3-mini",
+    "created_at": "2025-03-17T11:52:00+02:00",
+    "display_name": "O3-Mini",
+    "provider": "openai",
+    "context_window": 200000,
+    "max_tokens": 8192,
+    "type": "chat",
+    "family": "o3_mini",
+    "temperature": false,
+    "supports_vision": false,
+    "supports_functions": true,
+    "supports_json_mode": false,
+    "input_price_per_million": 1.10,
+    "output_price_per_million": 4.40,
+    "metadata": {
+      "object": "model",
+      "owned_by": "system"
+    }
+  },
```

Comment:
I added this manually, we would need to update `lib/tasks/models.rake` but I don't have all credentials to do so